cobra/core/Model.py

```diff
@@ -118,6 +117,45 @@ def description(self, value):
         self.name = value
         warn("description deprecated", DeprecationWarning)
 
+    @property
+    def medium(self):
+        return {rxn.id: rxn.bounds for rxn in
```

Comment:
how will this be with all export reactions, I didn't quite get the conclusion from the discussion, I note that  e.g. oleate export from yeast will show in up in the medium which perhaps isn't so intuitive?